package.json

```diff
@@ -161,28 +153,38 @@
     "not Safari < 10",
     "not ios_saf < 10"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "@babel/polyfill": "^7.10.4",
+    "core-js": "^3.6.5",
+    "regenerator-runtime": "^0.13.7"
+  },
```

Comment:
The reason I had `core-js-bundle and `regenerator-runtime` in devDependencies and bundled these through a copy routine, is so that our demo would work on GitHub Pages (since it doesn't install npm dependencies).

There should be no need for `@babel/polyfill`, as that is deprecated by Babel and its replacements are already present (core-js and regenerator-runtime).

Reply:
I think it is simpler and cleaner to have the demo only in dist (and we can copy or bundle dependencies during the build).

Reply:
I don't know for sure that GitHub Pages can allow any build process of its own.

Reply:
To my knowledge, the build needs to be done outside GitHub.

Reply:
Ok, I guess the important thing is just that the `dist`, including copies or bundled deps, is not gitignored. Then, yeah, that is simpler and cleaner.

Reply:
However, since it has to be bundled in `dist`, that brings us back to the fact that these should be moved back to `devDependencies`. Then regular users won't have to install these packages (they'll just get the `dist` copies installed--not all the `node_modules`).